wagtail/project_template/home/tests.py

```diff
@@ -0,0 +1,39 @@
+# home/tests.py
+
+from django.test import TestCase
```

Comment:
I think we should try to use our documented testing approach in https://docs.wagtail.org/en/stable/advanced_topics/testing.html


```suggestion
from wagtail.test.utils import WagtailPageTestCase
```

Also, have a review of that page for some other tips/practice we recommend in our docs.